WebApiCoreSeed.Domain/Services/UserService.cs

```diff
@@ -29,10 +31,47 @@ public UserService(WebApiCoreSeedContext dbContext)
         }
 
 
-        //public async Task<List<User>> GetAsync(Guid userId)
-        //{
-        //    return await _dbContext.Users;
-        //}
+        public async Task<List<User>> GetAsync()
+        {
+            var Users = _dbContext.Users.ToList();
+            return Users;            
+        }
+
+        public async Task<string> CreateAsync(User user)
+        {
+            string result = string.Empty;
+            _dbContext.Users.Add(user);
+            _dbContext.SaveChanges();
+            result = CodeResponse.Sucessful.ToString();
+            return result;
+
+        }
+
+        public async Task<string> UpdateAsync(User user)
+        {
+            string result = string.Empty;            
+            var userUpdate = (from u in _dbContext.Users
+                              where u.Id == user.Id
+                              select u).FirstOrDefault();
+
+            userUpdate.FirstName = user.FirstName;

```

Comment:
Just in case, you should check if `userUpdate` is null and return Failure in that case. Otherwise you may have an exception here